src/main/webapp/app/shared/circular-progress-bar/circular-progress-bar.component.scss

```diff
@@ -113,7 +113,7 @@ Due to the split circle of progress-left and progress right, we must use the ani
     $stepName: ($i * math.div(100, $howManySteps));
 
     //animation only the left side if below 50%
-    @if $i <= ($howManySteps/2) {
+    @if $i <= math.div($howManySteps, 2) {
```

Comment:
I think you need to import the math library in the beginning of the file. I also fixed this issue in some other branch (which does not have a PR yet)